test/lib/import/colorectal/providers/nhs_england/adhoc_handler_colorectal_test.rb

```diff
@@ -0,0 +1,33 @@
+require 'test_helper'
+
+class AdhocHandlerColorectalTest < ActiveSupport::TestCase
+  def setup
+    @record = build_raw_record('pseudo_id1' => 'bob')
+    @genocolorectal = Import::Colorectal::Core::Genocolorectal.new(@record)
+    @importer_stdout, @importer_stderr = capture_io do
+      @handler = Import::Colorectal::Providers::NhsEngland::AdhocHandlerColorectal.new(EBatch.new)
+    end
+    @logger = Import::Log.get_logger
+  end
+
+  test 'assign_variant_details' do
+    fake_record = build_raw_record('pseudo_id1' => 'bob')
```

Comment:
Not sure `fake_record` is a good name here?

Reply:
Updated in latest commit 🙏